packages/ui/src/components/ListV2/__stories__/Selectable.stories.tsx

```diff
@@ -1,7 +1,84 @@
-import type { ComponentStory } from '@storybook/react'
-import type { List } from '..'
-import { Template } from './Template.stories'
+import type { Story } from '@storybook/react'
+import { useMemo, useState } from 'react'
+import { List } from '..'
+import { ActionBar } from '../../ActionBar'
+import { Button } from '../../Button'
+import { Stack } from '../../Stack'
+import { Text } from '../../Text'
+import { columns, data as sourceData } from './resources'
 
-export const Selectable: ComponentStory<typeof List> = Template.bind({})
+export const Selectable: Story = () => {
+  const [data, setData] = useState(sourceData)
+  const [selectedIds, setSelectedIds] = useState<string[]>([])
 
-Selectable.args = { ...Template.args, isSelectable: true }
+  const selectedPlanets = useMemo(
+    () => data.filter(planet => selectedIds.includes(planet.id)),
+    [data, selectedIds],
+  )
+
+  const onDelete = (planetIds: string[]) => {
+    setData(currentPlanets =>
+      currentPlanets.filter(planet => !planetIds.includes(planet.id)),
+    )
+  }
+
+  return (
+    <>
+      <List
+        selectedIds={selectedIds}
+        onSelectedIdsChange={setSelectedIds}
+        columns={columns}
+      >
+        <List.Body>
+          {data.map(planet => (
+            <List.Row
+              key={planet.id}
+              id={planet.id}
+              checkboxDisabled={planet.id === 'home-sweet-home'}
+              checkboxTooltip={
+                planet.id === 'home-sweet-home'
+                  ? "Earth isn't selectable"
+                  : undefined
+              }
+            >
+              <List.Cell>
+                {planet.id === 'home-sweet-home'
+                  ? `${planet.name} (Not selectable)`
+                  : planet.name}
+              </List.Cell>
+              <List.Cell>{planet.perihelion}AU</List.Cell>
+              <List.Cell>{planet.aphelion}AU</List.Cell>
+            </List.Row>
+          ))}
+        </List.Body>
+      </List>
+      {selectedPlanets.length !== 0 ? (
+        <ActionBar>
+          <Stack
+            direction="row"
+            justifyContent="space-between"
+            alignItems="center"
+            gap={2}
+          >
+            <Text as="p" variant="bodySmall">
+              Planets selected :{' '}
+              {selectedPlanets.map(planet => planet.name).join(', ')}
+            </Text>
+            <Button
+              action
+              icon="delete"
+              onClick={() => onDelete(selectedIds)}
+            />
+          </Stack>
+        </ActionBar>
+      ) : null}
```

Comment:
Can we add a height and a padding in order to have a clear action bar:

![Screenshot 2023-01-18 at 10 34 48](https://user-images.githubusercontent.com/15812968/213135633-6b5ce6ca-a6eb-46f6-96d4-23279876be6e.png)

```suggestion
          <Stack
            direction="row"
            justifyContent="space-between"
            alignItems="center"
            gap={2}
            style={{ height: 100%, padding: '0 8px'}}
          >
            <Text as="p" variant="bodySmall">
              Planets selected :{' '}
              {selectedPlanets.map(planet => planet.name).join(', ')}
            </Text>
            <Button
              action
              icon="delete"
              onClick={() => onDelete(selectedIds)}
            />
          </Stack>
        </ActionBar>
      ) : null}
```